service/src/test/java/greencity/ModelUtils.java

```diff
@@ -2220,7 +2175,10 @@ public static Bag getTariffBag() {
             .createdBy("Taras Ivanov")
             .location(getLocation())
             .minAmountOfBags(MinAmountOfBag.INCLUDE)
-            .bagTranslations(getBagTransaltion()).build();
+            .description("Description")
+            .descriptionEng("DescriptionEng")
+            .name("Test")
+            .nameEng("a").build();
```

Comment:
I think `name` and `nameEng` are better

Reply:
I didn't catch that, Should I to delete description and descriptionEng setter?

Reply:
I see you are using `Test` value for the `name` and `a` value for the `nameEng`
My suggestion is to use `name` and `nameEng` values instead of `Test` and `a`